apps/ccf-eui/src/app/modules/ontology-exploration/ontology-tree/ontology-tree.component.scss

```diff
@@ -1,145 +1,72 @@
-    }
+  mat-tree {
+    --mat-tree-container-background-color: var(--sys-surface-container-low);
+    padding-left: 0.75rem;
   }
-  .node-container {
+
+  mat-tree-node {
     display: flex;
+    flex-direction: column;
+    padding-right: 0.75rem;
+    margin-left: -0.25rem;
   }
 
-  .biomarkers-toggle::ng-deep {
-    margin: 0.5rem;
-    padding-left: 1rem;
+  .node-content {
+    padding-left: 0;
+    padding-right: 2px;
   }
 
-  .node {
-    min-height: 0;
-    font-size: 1rem;
-    margin-bottom: 0.25rem;
-
-    .slider {
-      width: 100%;
-      transition-duration: 0.25s;
-      transition-timing-function: ease-in-out;
-      transition-property: width;
-      position: relative;
-      z-index: 1;
-
-      &.hidden {
-        width: 0;
-        ccf-opacity-slider {
-          display: none;
-        }
-      }
-
-      ::ng-deep ccf-opacity-slider {
-        height: 1.5rem;
-        margin-left: 1rem;
-
-        .slider-box {
-          width: 100%;
-
-          .slider-and-label {
-            width: 100%;
-          }
-        }
-
-        .mat-slider {
-          height: 1.5rem;
-          width: 15rem;
-
-          .mat-slider-wrapper {
-            top: 12px;
-          }
-        }
-
-        .opacity-value {
-          width: 3rem;
-        }
-      }
-    }
-
-    .num-results {
-      margin-right: 0.25rem;
-    }
-
-    &.inner-node {
-      button {
-        &.hidden {
-          display: none;
-        }
-      }
-    }
-
-    .opacity {
-      position: relative;
-      min-width: 1.5rem;
-    }
-
-    .toggle {
-      width: 1.5rem;
-      height: 1.5rem;
-      line-height: normal;
-      padding: 0;
-    }
-
-    .autocomplete-open .toggle {
-      border-bottom: none;
-    }
-
-    .non-expandable {
-      margin-left: 1.5rem;
-    }
-
-    .text {
-      cursor: pointer;
-      margin-left: 1rem;
-      opacity: 1;
-      transition-duration: 0.25s;
-      transition-timing-function: ease-in-out;
-      transition-property: opacity, width;
+  .node-label {
+    padding-left: 0.375rem;
+  }
 
-      &.hidden {
-        opacity: 0;
-        width: 0%;
-      }
+  .num-results,
+  .header-num-results {
+    margin-left: auto;
+  }
 
-      &.unavailable {
-        pointer-events: none;
-      }
-    }
+  .biomarkers-toggle {
+    --mat-standard-button-toggle-selected-state-text-color: var(--sys-secondary);
+    display: flex;
+    align-items: center;
+    min-height: 2.25rem;
+    padding: 0 1rem;
 
-    .num-results {
-      margin-left: auto;
+    .mat-button-toggle-group {
+      width: 100%;
     }
   }
 
-  .block {
-    display: block;
+  ::ng-deep .mat-mdc-button-touch-target {
+    height: 1.5rem;
+    width: 1.5rem;
   }
-}
-
-.scroll-gradient {
-  position: absolute;
-  height: 3rem;
-  width: 90%;
-  bottom: 0;
-  pointer-events: none;
 
-  &.hidden {
-    display: none;
+  ::ng-deep .hra-scroll-overflow-fade-gradient-bottom {
+    --_hra-scroll-overflow-fade-color: var(--sys-surface-container-low);
```

Comment:
Set `--hra-scroll-overflow-fade-color` instead of the one that starts with an underscore. That one is private. Also you shouldn't need ng-deep. Just put it in the host and it will be inherited by the scroll bar.